config/config.go

```diff
@@ -79,6 +80,7 @@ func Get() (*Config, error) {
 		DatasetAPIURL:              "http://localhost:22000",
 		FilterAPIURL:               "http://localhost:22100",
 		PopulationTypesAPIURL:      "http://localhost:27300",
+		ExternalPrefixURL:          "http://localhost:22000",
```

Comment:
Is this value always likely to be the same as that for the DatasetAPIURL? If so then perhaps the DatasetAPIURL could be used instead?